src/app/modules/item/components/item-children-edit-form/item-children-edit-form.component.html

```diff
@@ -1,11 +1,17 @@
 <ng-container *ngIf="itemData">
-  <alg-item-children-edit
-    #childrenEdit
-    (childrenChanges)="updateItemChanges($event)"
-    [itemData]="itemData"
-    [ngClass]="{disabled: disabled}"
-  ></alg-item-children-edit>
+  <ng-container *ngIf="['all','all_with_grant', 'children'].includes(itemData.item.permissions.canEdit); else notAuthorized">
```

Comment:
Since the PR #1096 refactoring the permissions, we should use the new helpers. Here to me it's `allowsEditingChildren` located in `src/app/shared/models/domain/item-edit-permission.ts`

Reply:
Fixed